### PR TITLE
chore: meta & cleanup

### PR DESCRIPTION
**Pull request by Favna**
Did someone order one dosage of GitHub meta and cleanup?

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+2. Replace the value of `REACT_APP_CLIENT_ID` to your bot's ID
+3. Go to the Discord developer portal: https://discordapp.com/developers/applications/
+4. Select your bot application and go to the `OAuth2` tab
+5. Add redirect URL's for `http://localhost:3000/oauth/callback` and `http://localhost:3000/oauth/guild`
+6. Run a local instance of Skyra, be sure to follow it's local development guidelines as well.
+
+-   You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
+
+6. Install dependencies with `yarn`.
+7. Run `yarn start` to start the React dev server
```
Comment (kyranet) [inline .github/CONTRIBUTING.md]: 7.
8.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+**The issue tracker is only for issue reporting or proposals/suggestions. If you have a question, you can find us in our [Discord Server](https://skyra.pw/join)**.
```
Comment (kyranet) [inline .github/CONTRIBUTING.md]:
```suggestion
**If you have a question, you can find us in our [Discord Server](https://skyra.pw/join)**.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+1. Duplicate `.env.development` and rename it to `.env.development.local`
+2. Replace the value of `REACT_APP_CLIENT_ID` to your bot's ID
+3. Go to the Discord developer portal: https://discordapp.com/developers/applications/
+4. Select your bot application and go to the `OAuth2` tab
+5. Add redirect URL's for `http://localhost:3000/oauth/callback` and `http://localhost:3000/oauth/guild`
+6. Run a local instance of Skyra, be sure to follow it's local development guidelines as well.
+
+-   You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
+
+6. Install dependencies with `yarn`.
```
Comment (Favna) [inline .github/CONTRIBUTING.md]:
```suggestion
7. Install dependencies with `yarn`.
```

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+After forking/cloning this repo to a folder locally:
+
+1. Duplicate `.env.development` and rename it to `.env.development.local`
+2. Replace the value of `REACT_APP_CLIENT_ID` to your bot's ID
+3. Go to the Discord developer portal: https://discordapp.com/developers/applications/
+4. Select your bot application and go to the `OAuth2` tab
+5. Add redirect URL's for `http://localhost:3000/oauth/callback` and `http://localhost:3000/oauth/guild`
+6. Run a local instance of Skyra, be sure to follow it's local development guidelines as well.
+
```
Comment (Favna) [inline .github/CONTRIBUTING.md]:
```suggestion
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+-   You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
```
Comment (Favna) [inline .github/CONTRIBUTING.md]:
```suggestion
  - You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
```

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (Favna) [inline .github/CONTRIBUTING.md]:
```suggestion
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+6. Install dependencies with `yarn`.
+7. Run `yarn start` to start the React dev server
```
Comment (Favna) [inline .github/CONTRIBUTING.md]:
```suggestion
8. Run `yarn start` to start the React dev server
```

File: README.md [hunk 8]
```diff
@@ -1,12 +1,49 @@
...
-1. Navigate to `src/meta/constants.js` and change `CLIENT_ID` to the bot you're using.
-2. Install dependencies with `yarn`.
-3. `yarn start`
+1. Duplicate `.env.development` and rename it to `.env.development.local`
+2. Replace the value of `REACT_APP_CLIENT_ID` to your bot's ID
+3. Go to the Discord developer portal: https://discordapp.com/developers/applications/
+4. Select your bot application and go to the `OAuth2` tab
+5. Add redirect URL's for `http://localhost:3000/oauth/callback` and `http://localhost:3000/oauth/guild`
+6. Run a local instance of Skyra, be sure to follow it's local development guidelines as well.
+
```
Comment (Favna) [inline README.md]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,12 +1,49 @@
...
+-   You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
```
Comment (Favna) [inline README.md]:
```suggestion
  - You have to set the CLIENT_ID, CLIENT_SECRET in Skyra's configuration to your own bot properly!
```

File: README.md [hunk 10]
```diff
@@ -1,12 +1,49 @@
...
+
```
Comment (Favna) [inline README.md]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -1,12 +1,49 @@
...
+6. Install dependencies with `yarn`.
```
Comment (Favna) [inline README.md]:
```suggestion
7. Install dependencies with `yarn`.
```

File: README.md [hunk 12]
```diff
@@ -1,12 +1,49 @@
...
+7. Run `yarn start` to start the React dev server
```
Comment (Favna) [inline README.md]:
```suggestion
8. Run `yarn start` to start the React dev server
```

File: package.json [hunk 13]
```diff
@@ -41,23 +37,46 @@
...
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [
+			"yarn lint",
+			"git add"
+		]
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/kyra/skyra.pw.git"
```
Comment (kyranet) [inline package.json]:
```suggestion
		"url": "git+https://github.com/kyranet/skyra.pw.git"
```

File: package.json [hunk 14]
```diff
@@ -41,23 +37,46 @@
...
+	},
+	"bugs": {
+		"url": "https://github.com/kyra/skyra.pw/issues"
```
Comment (kyranet) [inline package.json]:
```suggestion
		"url": "https://github.com/kyranet/skyra.pw/issues"
```

**Pull request merged**